src/docs/styles/layout/_layout.scss

```diff
@@ -49,6 +49,15 @@ body {
         margin-top: space(3x);
       }
     }
+
```

Comment:
i think we'll want to take another look at reducing the specificity of the layout overall and removing the :not on L47 rather than adding selectors to override other images. let's chat!